security/namespace-policy.md

```diff
@@ -43,7 +43,28 @@ spec:
       ports:
         - 6379
 ```
-To allow ingress traffic from endpoints in other namespaces, use a **namespaceSelector** in the policy rule. A namespaceSelector matches one or more namespaces based on the labels that are applied on the namespace. In the following example, ingress traffic is also allowed from endpoints with **color: blue** in namespaces with **shape: circle**.
+
+To allow ingress traffic from endpoints in another namespace, select the `projectcalico.org/namespace` label in the policy rule.  In the following example, ingress traffic is also allowed from endpoints with **color: blue** in the **staging** namespace.
```

Comment:
Think that only works in a global network policy?  For namespaced policies (and now for global ones too), we have `namespaceSelector` that lets you match pods in other namespaces.

Reply:
There's a bit more detail in the network policy docs to cover selector precedence and scopes now: https://docs.projectcalico.org/reference/resources/networkpolicy#selectors

Reply:
yes, I agree - this shouldn't work. You'll need to use a namespace selector in a namespaced NP rule.